qemu/tests/cfg/nvdimm.cfg

```diff
@@ -27,3 +27,12 @@
 
     variants:
         - nvdimm_basic:
+
+        - nvdimm_emulate:
+            # Please update host kernel line first, such as add "memmap=1G!5G": 
+            # 1G is the size, 5G is the start address of host memory
+            start_vm = "no"
+            nv_backend = "/dev/pmem0"
+            mem-path = ${nv_backend}
+            del pre_command
+            del post_command
```

Comment:
It looks the feature is only supported on pc (and maybe q35), can you please create a restriction?